tests/layer_tests/common/tf_layer_test_class.py

```diff
@@ -1,14 +1,18 @@
 # Copyright (C) 2018-2024 Intel Corporation
 # SPDX-License-Identifier: Apache-2.0
 
-from common.layer_test_class import CommonLayerTest
+import numpy as np
+from common.layer_utils import InferAPI
+from common.utils.common_utils import generate_ir_python_api
+from common.utils.tf_utils import save_to_pb
 from common.utils.tf_utils import summarize_graph
-
 from common.utils.tflite_utils import get_tflite_results, save_pb_to_tflite
-from common.utils.tf_utils import save_to_pb
+from pathlib import Path
+
 
+class CommonTFLayerTest:
```

Comment:
Why can't we keep inheritance from the `CommonLayerTest`?

Reply:
That is because output type and shape check is absent there. If I add it in the common part, some bugs in other frontends are revealed as well. So I decided to implement it for TF only.